applications/projects/runSofa/Main.cpp

```diff
@@ -418,7 +420,21 @@ int main(int argc, char** argv)
         loadVerificationData(verif, fileName, groot.get());
     }
 
+    if( computationTimeSampling<1 )
+    {
+        sofa::helper::AdvancedTimer::setEnabled("Init", true);
+        sofa::helper::AdvancedTimer::setInterval("Init", 1);
+        sofa::helper::AdvancedTimer::setOutputType("Init", computationTimeOutputType);
+        sofa::helper::AdvancedTimer::begin("Init");
```

Comment:
There is a duplicated begin("Init") here

Reply:
There was a bug in the advanced timer (variable initialization at -1 instead of 0), that I just removed. Should be fine but it is good if that is double checked.